pkg/diagnose/connectivity/core_endpoint.go

```diff
@@ -100,6 +100,20 @@ func Diagnose(diagCfg diagnosis.Config) []diagnosis.Diagnosis {
 				}
 				diagnoses = append(diagnoses, d)
 			}
+
+			// Send a request to the flare endpoint
+			flareEndpointURLMaker := func() string {
+				// Create flare endpoint to the shape of "https://<version>-flare.agent.datadoghq.com/support/flare"
+				url := utils.GetInfraEndpoint(config.Datadog)
+				baseURL, _ := utils.AddAgentVersionToDomain(url, "flare")
+				return baseURL + "/support/flare"
+			}
+			flareEndpointURL := flareEndpointURLMaker()
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Why do you need that function ? It's only called once, directly after its definition, I think you can just inline it

Reply:
Yes, I just thought it would make code more easy to read, however I can inline it yes